core/store/src/genesis/initialization.rs

```diff
@@ -57,6 +69,7 @@ pub fn initialize_sharded_genesis_state(
         let mut store_update = store.store_update();
         set_genesis_hash(&mut store_update, &genesis_hash);
         set_genesis_state_roots(&mut store_update, &state_roots);
+        set_genesis_height(&mut store_update, &genesis.config.genesis_height);
```

Comment:
mini nit: Maybe worth refactoring it? It's the same in a few places. Not a biggie, feel free to ignore.

Reply:
Refactoring it how exactly?

Reply:
I was thinking about moving it to a helper method.